src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -410,7 +410,8 @@ void checkConstraintLanguageVersionAndRuntimeSupportForConversion(SyntaxNode syn
                 }
                 else if (conversion.IsInlineArray)
                 {
-                    // PROTOTYPE(InlineArrays): Check language version and runtime support
+                    CheckFeatureAvailability(syntax, MessageID.IDS_FeatureInlineArrays, diagnostics);
```

Comment:
Should the PROTOTYPE comment about checking for runtime support remain?

Reply:
> Should the PROTOTYPE comment about checking for runtime support remain? (also applies elsewhere)

Yes, language version check is orthogonal to runtime support.

Reply:
Right, so I'd think the comment about runtime support should have remained since it wasn't implemented.

Reply:
> Right, so I'd think the comment about runtime support should have remained since it wasn't implemented.

I see what you mean. I'll put it back. Thanks.